finalPrep/README.md

```diff
@@ -18,37 +18,40 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+<img width="1194" alt="Screen Shot 2021-07-24 at 10 03 37 PM" src="https://user-images.githubusercontent.com/87044013/126887412-bd2faf6e-372f-4a70-9102-6b3a489bfaf6.png">
```

Comment:
Looks like a good start! I say that, because your calendar is always a work in progress.
A few things to think about:
- Weekends are sparse at the moment. Project work and review will become parts of your weekends, so you may think about blocking work time versus free time. This will could change as you get a better feel for Turing life, but do plan on spending time working on the weekends. It is important to get those social items in your calendar as well. 
- That's a pretty late dinner for such an early morning. No suggestion here, just surprised that your dinner was quite a bit later than your wind-down time.
- You should consider taking the same approach with your homework blocks as you do with your workout blocks- like, what specific thing or 2 will I do with that time? This can help you prioritize and focus.